content/en/docs/components/pipelines/author-a-pipeline/components.md

```diff
@@ -164,7 +164,77 @@ To create a containerized component, you must:
 **When to use?** Containerized Python components should be used any time your component is implemented as Python code, but cannot be written as a standalone Python function or you wish to organize source code outside of the component Python function definition.
 
 ### 3. Custom container components
-<!-- @zichuan-scott-xu to provide this section -->
+
+Container-based components make it easier to use the pre-existed container to build a KFP component. 
+The user will control the container entrypoint and is able to define the component by specifying their container, commands, and args in a `ContainerSpec` object with similar authoring experiences as defining Python components. 
```

Comment:
This document addresses the reader directly, saying "you".

Can you update your doc to take this style? E.g., `You can control the container entrypoint`